### ISSUE TITLE
Fix Data Preview- 508 Issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1493]
## Required Fix 

- [x] RF-6
- [x] RF-9
- [x] RF-14 

## Context 
See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

| Issue #  | Issue Description  |
|-|-|
| RF-6 | There is a button present that is   lacking information which a user needs  to understand its purpose. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 4.1.2 |
| RF-9 | Form fields that are grouped together, such as radio buttons and check boxes, do not have fieldset and legend tags. These tags provide understandability for assistive technology users. This issue is a violation of WCAG 2.0 Success Criterion 1.3.1 |
| RF-14 | An actionable element on a web page triggers an update or adds new content with no change in URL, but there is no method to alert Assistive Technology users of the update. This issue is a violation of Section 508 and WCAG 2.0 Success Criterion 1.3.1. |

## Testing 
Tester: @aprematta

**Comment by aprematta** [comment]
One issue found with RF-6, the rest pass:


RF-6 - 
All Close buttons are labeled close instead of Close <filter>. I would actually say remove instead of close. So 'Remove Filter for xxxxx"

**Comment by aprematta** [comment]
Testing complete, closing ticket
